notebook_lifecycle_scripts_examples/install_pyspark_conda_create_notebook.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+#This script allows you to install pyspark conda environment to work with data flow in data science notebook session. It creates a ipynb  file that imports ads and loads the SparkMagic extension
+
+# enable conda commands
+source /etc/profile.d/enableconda.sh
+
+ENV_NAME=pyspark32_p38_cpu_v3
+
+# Path to the conda environment folder
+ENV_FOLDER="$HOME/conda"
+
+# Check if the conda environment exists
+if [ -d "$ENV_FOLDER/$ENV_NAME" ]
+then
+    echo "Conda environment '$ENV_NAME' found."
+
+else
+    echo "Conda environment '$ENV_NAME' not found, installing..."
+    odsc conda install -s "$ENV_NAME"
+fi
+
+# Activate the conda environment
+conda activate "$ENV_FOLDER"/"$ENV_NAME"
+
+echo "Conda environment '$ENV_NAME' is now activated."
+
+
+cat << EOF > pyscript.py
```

Comment:
Should we just create a Python script instead of creating a python file through bash script?

Reply:
I looked into combining everything into one python script but it looks difficult to activate the conda env and then run the python code needed to create the ipynb notebook